guides/v2.3/graphql/product/product-interface.md

```diff
@@ -98,6 +98,7 @@ Attribute |  Data Type | Description
 --- | --- | ---
 `discount` | ProductDiscount | The amount of the discount applied to the product. It represents the difference between the `final_price` and `regular_price`
 `final_price`| Money! | The price of the product after applying discounts
+`fixed_product_taxes` | [[FixedProductTax](#FixedProductTax)] | An array of fixed product taxes applied or can be applied to a product price
```

Comment:
I kept stumbling on this sentence and wanted to add some clarifying text. But feel free to STET.
```suggestion
`fixed_product_taxes` | [[FixedProductTax](#FixedProductTax)] | An array of fixed product taxes either applied or that can be applied to a product price
```